blocks/email-composer/email-composer.js

```diff
@@ -187,64 +208,54 @@ export default async function decorate(block) {
         }
       };
 
-      // Load site to retrieve drive id
-      fetch(`${SCRIPT_API}/list/${id}`, {
-        headers: {
-          authorization: `bearer ${token}`,
-        },
-      }).then((res) => {
-        if (res.ok) {
-          return res.json();
-        }
-
-        throw new Error(res.status);
-      })
-        .then(async (res) => {
-          project = res.project;
-
-          block.querySelector('.actions').innerHTML = `
+      block.querySelector('.actions').innerHTML = `
             <a href="#" target="_blank" class="button secondary action copy">Copy</a>
-            <a href="${project.driveUrl}" target="_blank" class="button action secondary edit">Edit</a>
+            <a href="${project?.driveUrl || `https://da.live/edit#/da-self-service/${id}/newsletter`}" target="_blank" class="button action secondary edit">Edit</a>
```

Comment:
I dont think you can hardcode newsletter here

Reply:
I removed it